### PR TITLE
Group chatbot messages by default

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Closes: #8850

By default, consecutive messages are displayed in the same bubble. This is controlled by the new `display_consecutive_in_same_bubble` param of Chatbot.

consecutive_messages

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Testing and Formatting Your Code

1. PRs will only be merged if tests pass on CI. We recommend at least running the backend tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the backed tests: `bash scripts/run_backend_tests.sh`

2. Please run these bash scripts to automatically format your code: `bash scripts/format_backend.sh`, and (if you made any changes to non-Python files) `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/components/chatbot.py [hunk 1]
```diff
@@ -198,6 +198,7 @@ def __init__(
         examples: list[ExampleMessage] | None = None,
         show_copy_all_button=False,
         allow_file_downloads=True,
+        display_consecutive_in_same_bubble: bool = True,
```
Comment (abidlabs) [inline gradio/components/chatbot.py:201]: Nit: the parameter name is kinda long. How about `group_consecutive_messages`?

**Comment by abidlabs** [comment]
The only small thing I noticed was that when a file is not the first message in a group of messages, then the top right icons look "off" because they are rounded and lack a top border. E.g. look at the icons in the top right of the audio waveform here:

image

(Produced by running `python demo/test_chatinterface_multimodal_examples/run.py` and selecting the rightmost example.)

Otherwise, LGTM!

**Comment by freddyaboulton** [comment]
Should be good for a re-review now - thanks @hannahblair for fixing the styling. We have omitted borders but fixed the rounding. We tried the borders but we think it looks better without.

**Review by abidlabs — APPROVED** [review]
Tested and looks great! Only nit is about shortening the parameter name but otherwise good to me

**Pull request merged**